Integrations/integration-Cybereason.yml

```
@@ -518,6 +518,12 @@ script:
                 machine_name = machine.get('name', '')
                 affected_machines.append(machine_name)
 
+            involved_hashes = []
+            if 'rootCauseElementHashes' in malop['simpleValues']:
+                if malop['simpleValues']['rootCauseElementHashes']['totalValues'] != 0:
+                    for hashcake in malop['simpleValues']['rootCauseElementHashes']['values']:
```

no need for the loop
```suggestion
                   involved_hashes.extend(malop['simpleValues']['rootCauseElementHashes']['values'])
```